src/main/java/org/tron/common/overlay/discover/DiscoverServer.java

```diff
@@ -47,14 +48,15 @@
 
   Args args = Args.getInstance();
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Explicitly declare the visibility for "args". ([squid:S2039](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2039))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=951&issues=AWPTJmwkA0k93xG_IQRa&open=AWPTJmwkA0k93xG_IQRa)

[ISSUE_KEY]: AWPTJmwkA0k93xG_IQRa